runauto_nag_weekdays.sh

```diff
@@ -187,6 +187,9 @@ python -m auto_nag.scripts.fuzz_blockers --production
 # Drop old severities
 python -m auto_nag.scripts.severity_migration --production
 
+# Detect bugs with small crash volume
+python -m auto_nag.scripts.crash_samll_volume --production
```

Comment:
```suggestion
python -m auto_nag.scripts.crash_small_volume --production
```

Reply:
Don in 526eca30fe7bd70416fa3ee2a278deff4fe657f8.